e2e/tasks/deleteDownloads.js

```diff
@@ -3,16 +3,21 @@ const path = require('path')
 
 function deleteDownloads(config) {
```

Comment:
I have no easy way to check on the phone: Do we still use deleteDownloads at all now? I think we do, because we paranoidly delete everything before running a download sensitive e2e test, right?

Reply:
yes, we still delete the downloads before we download the pdfs.
But instead of deleting them at the end of the tests, we move them now than we can upload them in to the job build artifacts.